templates/laravel/files/.platform.app.yaml

```diff
@@ -39,7 +39,8 @@ hooks:
     deploy: |
         set -e
         php artisan optimize:clear
-
+        php artisan optimize
+        php artisan view:cache
```

Comment:
I though view:cache was/would have been bundled in optimize.

Reply:
In testing yesterday, view cache is not repopulated into `storage/framework/views` after a `php artisan optimize` run. 

```
web@app.0:~$ ls -al storage/framework/views/
total 24
drwxr-xr-x 2 web web  4096 Feb 14 23:36 .
drwxr-xr-x 6 web web    90 Oct 16 21:49 ..
-rw-r--r-- 1 web web 18607 Feb 14 23:36 011ac5c3fc8bb2fd56c323f5510bae609400be4b.php
```
views has content

```
web@app.0:~$ php artisan optimize:clear
Cached events cleared successfully.
Compiled views cleared successfully.
Application cache cleared successfully.
Route cache cleared successfully.
Configuration cache cleared successfully.
Compiled services and packages files removed successfully.
Caches cleared successfully.
```
optimize clear indicates it cleared view cache
```
web@app.0:~$ php artisan optimize
Configuration cache cleared successfully.
Configuration cached successfully.
Route cache cleared successfully.
Routes cached successfully.
Files cached successfully.
```
interestingly, `optimize` makes no mention of generating view cache
```
web@app.0:~$ ls -al storage/framework/views/
total 4
drwxr-xr-x 2 web web 4096 Feb 15 17:10 .
drwxr-xr-x 6 web web   90 Oct 16 21:49 ..
```
contents of views is empty
```
web@app.0:~$ php artisan view:cache
Compiled views cleared successfully.
Blade templates cached successfully.
web@app.0:~$ ls -al storage/framework/views/
total 76
drwxr-xr-x 2 web web  4096 Feb 15 17:10 .
drwxr-xr-x 6 web web    90 Oct 16 21:49 ..
-rw-r--r-- 1 web web 18607 Feb 15 17:10 011ac5c3fc8bb2fd56c323f5510bae609400be4b.php
-rw-r--r-- 1 web web  2589 Feb 15 17:10 0bdcc7cd443578dab284f75f8ba0bbd2b5374032.php
-rw-r--r-- 1 web web  1370 Feb 15 17:10 389c74d4998d0d702ffafb558ffc7d8eb2a99ffa.php
-rw-r--r-- 1 web web  5237 Feb 15 17:10 5a4dfa7c8528dd727e0dac392671535f591e1cd2.php
-rw-r--r-- 1 web web  1897 Feb 15 17:10 77b243c2dd7535055cf86e0ff5b053a2f6633fc1.php
-rw-r--r-- 1 web web  1066 Feb 15 17:10 8fd6e42198272f151485f31a4efb4c473d946602.php
-rw-r--r-- 1 web web  2813 Feb 15 17:10 abf3848f59b512a29d37d438ecfa16d1d809ece1.php
-rw-r--r-- 1 web web  8720 Feb 15 17:10 afc5cfc310ea2011b037259c963b7f8da85f086f.php
-rw-r--r-- 1 web web  2051 Feb 15 17:10 f1144557e9322c3af204a497e3e0b4daab32adfc.php
-rw-r--r-- 1 web web  2423 Feb 15 17:10 fe83f506edbb5429889bd400a38ae096ad2759db.php
-rw-r--r-- 1 web web  1350 Feb 15 17:10 fed726ca8dccaf5e4c9ae9a7f183c98b3b049cec.php
```
Now we have view cache again